documentation.md

```diff
@@ -0,0 +1,344 @@
+# Classroom Angel Internal API Documentation
+
+## Teacher Attendance
+
+- GET to [https://classroom-angel-development.herokuapp.com/api/attendance/](https://classroom-angel-development.herokuapp.com/api/attendance/) - retrieve all attendance records
+  URL Params | Body | Returns
+  --- | --- | ---
+  N/A | N/A | All attendance objects (return)
```

Comment:
This table looks like it isn't fully formatted!

Reply:
Done